README.md

```diff
@@ -9,6 +9,52 @@ submissions hold. Its evolution is coupled with the [hepdata-pyhf-visualizer][py
 JS package, as this one renders what gets summarized by this component.
 
 
+## Installation
+
+Via pip:
+```sh
+pip install hepdata-pyhf-extractor
+```
+
+Via GitHub (for developers):
+```
+git clone https://github.com/HEPData/hepdata-pyhf-extractor
+cd hepdata-pyhf-extractor
+pip install ".[dev]"
+```
+
+
+## Usage
+
+To extract basic information out of a `pyhf` Pallet file, the summarizers can:
+
+A) Receive a Pallet file path for them to open and read it internally.
+
+```python3
+import gzip
+import json
+from pyhf_extractor import V1PalletSummarizer
+
+pallet_path = "example_pallet.json"
+
+with gzip.open(pallet_path) as file:
+    pallet_data = json.load(file)           
+
+summarizer = V1PalletSummarizer(pallet_data=pallet_data)
+summary = summarizer.summarize()
+```
+
+B) Receive an already parsed Pallet file:
+```python3
+from pyhf_extractor import V1PalletSummarizer
+
+pallet_path = "example_pallet.json"
```

Comment:
Maybe comment that `pallet_path` can be either a `.json` or `.json.gz` file.

Reply:
Yes, we can expand the example.

Done in https://github.com/HEPData/hepdata-pyhf-extractor/pull/6/commits/d28ef110c6c64fdf40313cdd488c577094410619